src/modules/fancyzones/FancyZonesLib/WindowMoveHandler.cpp

```diff
@@ -59,24 +59,24 @@ WindowMoveHandler::WindowMoveHandler(const winrt::com_ptr<IFancyZonesSettings>&
 {
 }
 
-void WindowMoveHandler::MoveSizeStart(HWND window, HMONITOR monitor, POINT const& ptScreen, const std::unordered_map<HMONITOR, winrt::com_ptr<IWorkArea>>& zoneWindowMap) noexcept
+void WindowMoveHandler::MoveSizeStart(HWND window, HMONITOR monitor, POINT const& ptScreen, const std::unordered_map<HMONITOR, winrt::com_ptr<IWorkArea>>& workAreaMap) noexcept
 {
     if (!FancyZonesUtils::IsCandidateForZoning(window, m_settings->GetSettings()->excludedAppsArray) || WindowMoveHandlerUtils::IsCursorTypeIndicatingSizeEvent())
     {
         return;
     }
 
-    m_moveSizeWindowInfo.hasNoVisibleOwner = FancyZonesUtils::HasNoVisibleOwner(window);
-    m_moveSizeWindowInfo.isStandardWindow = FancyZonesUtils::IsStandardWindow(window);
-    m_inMoveSize = true;
+    m_activeWindowInfo.hasNoVisibleOwner = FancyZonesUtils::HasNoVisibleOwner(window);
+    m_activeWindowInfo.isStandardWindow = FancyZonesUtils::IsStandardWindow(window);
+    m_isActive = true;
 
-    auto iter = zoneWindowMap.find(monitor);
-    if (iter == end(zoneWindowMap))
+    auto iter = workAreaMap.find(monitor);
+    if (iter == end(workAreaMap))
     {
         return;
     }
 
-    m_windowMoveSize = window;
+    m_activeWindow = window;
```

Comment:
This could cause misunderstanding. The active window is generally the window that has focus, and in our case, it would be wrong. This is the window that is moving or resizing at the moment. I don't like the `MoveSize` in naming, but at least it's more clear.

Reply:
How about `current`?

Reply:
Do you prefer I'd revert the change?

Reply:
More options:
* `m_{window,windowInfo,workArea}`
* `m_subject{Window,WindowInfo,WorkArea}`
* `m_current{Window,WindowInfo,WorkArea}`
* `m_dragged{Window,WindowInfo,WindowWorkArea}`

Reply:
I like the `m_draggedWindow` :)

Reply:
And `m_isDragging`?